pkg/vere/serf.c

```diff
@@ -70,18 +72,61 @@ enum {
   _serf_fag_vega = 1 << 4   //  kernel reset
 };
 
+/* _serf_quac: convert a quac to a noun.
+*/
+u3_noun
+_serf_quac(quac* mas_u)
```

Comment:
Perhaps you could condense `_serf_quac` and `_serf_quacs` to a single function that returns just a `quac` noun or a `(list quac)` depending on the `len_w` parameter?

Reply:
I'm wary of a function that returns two different hoon types depending on its arg. Do you think that's ok?